ext/qiita_marker/html.c

```diff
@@ -222,6 +223,18 @@ static int S_render_node(cmark_html_renderer *renderer, cmark_node *node,
           escape_html(html, node->as.code.info.data + first_tag + 1, node->as.code.info.len - first_tag - 1);
         }
         cmark_strbuf_puts(html, "\"><code>");
+      } else if (options & CMARK_OPT_CODE_DATA_METADATA) {
```

Comment:
[memo]
CMARK_OPT > CODE_DATA_METADATA

Reply:
[memo]
`CODE_DATA_METADATA` is one of the `CMARK_OPT`.